graphistry/PlotterBase.py

```diff
@@ -38,6 +38,7 @@
 from .arrow_uploader import ArrowUploader
 from .nodexlistry import NodeXLGraphistry
 from .tigeristry import Tigeristry
+from .database_clients.spannergraph import spannergraph
```

Comment:
this top-level import would force all users to import, so we need to be lazy/dynamic somewhere